RecoParticleFlow/PFClusterTools/src/PFEnergyCalibration.cc

```diff
@@ -193,21 +200,26 @@ PFEnergyCalibration::energyEmHad(double t, double& e, double&h, double eta, doub
     t = min(999.9,max(tt, thresh+a*e+b*h));
     
     // The angular correction 
- 
     double dEta = std::abs( std::abs(eta) - 1.5 );
     double etaPow = dEta * dEta * dEta * dEta;
-    //MM: 0.1 factor helps the parametrization
-    etaCorrE = 1. + aEtaEndcap(t) + 0.1*bEtaEndcap(t)*etaPow;
-    etaCorrH = 1. + aEtaEndcap(t) + bEtaEndcap(t)*etaPow;
-    
+
+    if ( e > 0. && thresh > 0. ) {
+      etaCorrE = 1. + aEtaEndcapEH(t) + 1.3*bEtaEndcapEH(t)*(0.04 + etaPow);
+      etaCorrH = 1. + aEtaEndcapEH(t) + bEtaEndcapEH(t)*(0.04 + etaPow);
+    } else {
+      etaCorrE = 1. + aEtaEndcapH(t) + 1.3*bEtaEndcapH(t)*(0.04 + etaPow);
```

Comment:
here and for the barrel: which pages on the slides provided describe the magic numbers (as well as eta dependence change itself)? We have 1.3, 0.04, 0.05, and I didn't find a match in the slides in PPD